### PR TITLE
React router

### PR DESCRIPTION
**Pull request by moise-mulungu**
On this milestone, I:
- Installed react-router
- Created three more component files
- Added contents on each file
- Created a route for each page
- Executed my SPA on the browser

### CODE REVIEW

File: src/components/home.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Home = () => (
+  <div className="home-container">
+    <h1 className="home-heading">Welcome to our page!</h1>
+    <p className="home-para">We have made this web application to give you access to a magic calculator that you can use anytime, anywhere.</p>
+    <p className="home-para">You will use all possible basic Math operations such as Addition, Subtraction, Multiplication, Division, plus some advanced operations such as Percentage, Exponentiation and more...</p>
+    <p className="home-para">Explore them on the links above</p>
```
Comment (Emmyn5600) [inline src/components/home.js:8]: - [ ] Optional suggestion

   - Kindly, keep one `<p>` tag, with all the text, Imagine once you have more than 100 lines with the `<p>` tag. How would manage it?

File: src/components/quote.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Quote = () => (
+  <div>
+    <p className="quote">Mathematics is not something about numbers, equations, computaions,or algorithm: </p>
```
Comment (Emmyn5600) [inline src/components/quote.js:5]: - [x] Optional suggestion

   - Kindly, keep one `<p>` tag, with all the text, Imagine once you have more than 100 lines with the `<p>` tag.

**Review by Emmyn5600 — APPROVED** [review]
Hi @moise-mulungu 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! :tada: :clap: :clap: :100:

nodding-yes

## Optional suggestions

- [ ] N/A

##### _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

##### Cheers and Happy coding!:clap::clap::clap:
##### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/components/quote.js (hunk 2 above)
Comment (moise-mulungu) [reply src/components/quote.js:5]: Thank you, I will fix that

**Pull request merged**